mlflow/keras.py

```diff
@@ -82,9 +82,13 @@ def save_model(keras_model, path, conda_env=None, mlflow_model=Model(), custom_o
                                 'tensorflow=1.8.0'
                             ]
                         }
-    :param keras_module: Keras module to be used to save / load the model. If not provided, MLflow
-    will attempt to infer the Keras module based on the given model.
     :param mlflow_model: MLflow model config this flavor is being added to.
+    :param custom_objects: Keras ``custom_objects`` dictionary containing String:Class
+    mappings for custom layers in your Keras model. MLflow saves these custom layers using
```

Comment:
nit: I think we should indent mappings at the same level of `Keras`.

Reply:
+1!